README.md

```diff
@@ -36,27 +46,47 @@ We're creating the very network features we need to host events and gather well.
 
 ### Upcoming features
 
-- Allow attendees to add guests.
-- Import events from meetup.com with an add-on plugin (currently in development).
-- Recurring events.
-- Calendar block.
-- Email notification when event starts.
+- Allow attendees to add guests. ([#](https://github.com/GatherPress/gatherpress/issues/86))
+- Import events from meetup.com with an add-on plugin. ([#](https://github.com/GatherPress/gatherpress/issues/394))
+- Recurring events. ([#](https://github.com/GatherPress/gatherpress/issues/80))
+- Calendar block. ([#](https://github.com/GatherPress/gatherpress/issues/369))
+- Email notification when event starts. ([#](https://github.com/GatherPress/gatherpress/issues/429))
+- Event federation using ActivityPub ([#](https://github.com/GatherPress/gatherpress/issues/569))
+
+### Contribute
+
+If you wish to share in the collaborative of work to build GatherPress, please drop us a line either via [WordPress Slack](https://make.wordpress.org/chat/) or on [GatherPress.org](https://gatherpress.org/get-involved). The development location of the GatherPress project can be found at [https://github.com/gatherpress/gatherpress](https://github.com/gatherpress/gatherpress). All contributions are welcome: code, design, user interface, documentation, [translation](https://translate.wordpress.org/projects/wp-plugins/gatherpress/) and more.
+
+### Credits
+
+Thanks to the following folks for contributing to GatherPress:
+[mauteri](https://profiles.wordpress.org/mauteri/), [hrmervin](https://profiles.wordpress.org/hrmervin/), [pbrocks](https://profiles.wordpress.org/pbrocks/), [jmarx](https://profiles.wordpress.org/jmarx/), [prashantabellad](https://profiles.wordpress.org/prashantabellad/), [prayagm](https://profiles.wordpress.org/prayagm/), [patricia70](https://profiles.wordpress.org/patricia70/), [meaganhanes](https://profiles.wordpress.org/meaganhanes/), [calebthedev](https://profiles.wordpress.org/calebthedev/), [javiercasares](https://profiles.wordpress.org/javiercasares/), [linusx007](https://profiles.wordpress.org/linusx007/), [cameronbarrett](https://profiles.wordpress.org/cameronbarrett/), [courane01](https://profiles.wordpress.org/courane01/), [newyorkerlaura](https://profiles.wordpress.org/newyorkerlaura/), [phoopee3](https://profiles.wordpress.org/phoopee3/), [hauvong](https://profiles.wordpress.org/hauvong/), [deshabhishek007](https://profiles.wordpress.org/deshabhishek007/), [jeffpaul](https://profiles.wordpress.org/jeffpaul/), [michelleames](https://profiles.wordpress.org/michelleames/)
+
```

Comment:
Let's make this list just the folks listed as Leads and on Team. See Settings -> Credit. That will keep this list maintainable.

Reply:
I would like to see my handle in the list, too.